sirepo_bluesky/tests/test_srw_det.py

```diff
@@ -38,6 +41,9 @@ def _test_srw_detector(RE, db, tmpdir, sim_type, sim_id, server_name):
 
     assert mean == 1334615738479247.2, "incorrect mean value from bp.count"
 
+    sim_durations = np.array(db[-1].table()["srw_det_duration"])
```

Comment:
We already have a reference to the table - `t`, so we can reuse it:
```suggestion
    sim_durations = np.array(t["srw_det_duration"])
```